### PR TITLE
feat: Upgrade cozy dependencies for App Amirale navigation

### PR DESCRIPTION
**Pull request by acezard**
https://trello.com/c/KWxWhWBK/274-%F0%9F%8E%81-e162-gestion-des-navbar-et-statusbar-dans-lensemble-de-laa-4-jh

For statusbar and navbar to be handled in app amirale, cozy-intent and more importantly cozy-ui have to be updated on every cozy app

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,11 +52,11 @@
     "cozy-client": "^25.0.0",
     "cozy-device-helper": "^1.17.0",
     "cozy-doctypes": "^1.70.0",
-    "cozy-intent": "^1.9.1",
+    "cozy-intent": "^1.13.2",
     "cozy-interapp": "0.5.6",
     "cozy-keys-lib": "3.7.1",
     "cozy-realtime": "^3.13.1",
-    "cozy-ui": "51.11.0",
+    "cozy-ui": "^62.2.0",
```
Comment (trollepierre) [inline package.json:59]: broken pipeline

File: test/__snapshots__/app.spec.js.snap [hunk 2]
```diff
@@ -61,6 +61,6 @@ exports[`App component only should be mounted correctly 1`] = `
       />
     </Switch>
   </Main>
-  <IconSprite />
+  <UNDEFINED />
```
Comment (trollepierre) [inline test/__snapshots__/app.spec.js.snap:64]: According to :
https://github.com/cozy/cozy-ui/releases/tag/v52.0.0
 there is a Breaking change here

**Review by trollepierre — APPROVED** [review]

File: test/__snapshots__/app.spec.js.snap (hunk 2 above)
Comment (Crash--) [reply test/__snapshots__/app.spec.js.snap:64]: So you've to change the way you import the Sprite. Snapshot can be useful in fact :D

**Review by Crash-- — CHANGES_REQUESTED** [review]
You have to change the IconSprite import

**Review by trollepierre — APPROVED** [review]

**Review by trollepierre — APPROVED** [review]

File: test/__snapshots__/app.spec.js.snap (hunk 2 above)
Comment (acezard) [reply test/__snapshots__/app.spec.js.snap]: @Crash-- indeed. I updated.
I think cozy-settings wasn't updated in quite some time, it was far harder to update than the other cozy-apps (some of them did not have a single issue).

**Pull request merged**